### PR TITLE
Add EACEngine

### LINKED ISSUE #2393: [FEATURES]Add EACEngine
**What feature you'd like to add:**
Add EACEngine.

**Why is this feature needed:**

### PR DESCRIPTION
**Pull request by myccccccc**
### Ⅰ. Describe what this PR does
Add EACEngine

### Ⅱ. Does this pull request fix one issue?

fixes #2393

### Ⅲ. List the added test cases (unit test/integration test) if any, please explain if no tests are needed.
unit test

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: pkg/ddc/eac/dir_quota.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+package eac
+
+import (
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk"
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/auth/credentials"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/nas"
+)
+
+func (mountInfo MountInfo) SetDirQuota() (response *nas.SetDirQuotaResponse, err error) {
```
Comment (cheyang) [inline pkg/ddc/eac/dir_quota.go:25]: Why do we have to use opensdk to setDirQuota?

Comment (myccccccc) [reply pkg/ddc/eac/dir_quota.go]: > Why do we have to use opensdk to setDirQuota?

@cheyang Removed, we will include a setDirQuota/describeDirQuota cmd tool in later PRs.

**Review by cheyang — APPROVED** [review]

**Pull request merged**